browsers/firefox/manifest.json

```diff
@@ -55,6 +55,9 @@
           "run_at": "document_idle"
       }
     ],
+    "options_ui": {
+        "page": "html/options.html"
+  },
```

Comment:
Can you fix the indentation here?

Reply:
Not without a new PR as I removed the branch in the meantime: https://github.com/duckduckgo/duckduckgo-privacy-extension/pull/658

Hopefully it gets reviewed in less then a year this time ;)